services/headless-lms/server/src/programs/seed/seed_courses.rs

```diff
@@ -2204,6 +2204,277 @@ pub async fn seed_switching_course_instances_course(
+* The soroban is smaller, more streamlined, and taught in schools even today as a mental math tool.
+
+Japanese abacus masters, known as **soroban experts**, have been known to compete with electronic calculators—and sometimes win—in speed and accuracy contests.
+
+## The Russian Schoty: A Cultural Twist
+
+Another significant variant is the **Russian abacus**, or **schoty (счёты)**, developed in the 17th century. Unlike the Chinese and Japanese versions, the schoty is:
+
+* Used **horizontally**, not vertically.
+* Has **10 beads per wire** (except for one wire with four for quarter-ruble calculations).
+* Operated using **one hand**, usually the right.
+* Primarily used in Russian markets and accounting offices throughout the Soviet era.
+
+The schoty reflects a different approach to abacus use but remains a powerful and intuitive calculator for those trained in its methods.
+
+## The Abacus in the Modern Era
+
+By the 17th century, the invention of **mechanical calculators** (such as Pascal's calculator) and later **electronic calculators** in the 20th century began to replace the abacus in many parts of the world. However, the abacus never truly disappeared.
+
+### Contemporary Uses
+
+* In many Asian countries, especially **China, Japan, and Taiwan**, the abacus is still taught to schoolchildren.
+* It’s used to develop **mental arithmetic** skills, memory, and concentration.
+* Abacus competitions are held globally, where students perform lightning-fast calculations.
+* The abacus is also used as an educational tool for **visually impaired** students due to its tactile nature.
+
+## The Abacus as a Cultural Icon
+
+More than just a calculating device, the abacus has become a cultural symbol:
+
+* In **Chinese culture**, the abacus symbolizes **wisdom and financial acumen**.
+* In Japan, the soroban is associated with **discipline and mental strength**.
+* The image of the abacus is often used in logos and iconography related to **mathematics, finance, and education**.
+
+## Conclusion: The Legacy of the Abacus
+
+The abacus has endured for thousands of years, evolving across continents and cultures. From Mesopotamian merchants to modern-day students mastering mental math, it has remained a testament to human creativity and the universal need to understand and manipulate numbers.
+
+In an age where technology continues to advance at lightning speed, the abacus reminds us that sometimes, the simplest tools can be the most profound. Whether made of beads, wood, or stone, the abacus is a bridge between the ancient and the modern—an enduring legacy of humankind’s quest for knowledge.
+
+",
+                                cx.v5(b"page:2:1:block:intro"),
+                            ),
+                        )),
+                ),
+        );
+
+    let (course, _default_instance, _last_module) = course.seed(&mut cx).await?;
+
+    chatbot_configurations::insert(
```

Comment:
extend the builder api, and use same hardcoded ids for the chatbot configuration and the blocks that reference them